### DISCUSSION TITLE
Publicly share smart albums

### DISCUSSION THREAD

**Discussion by PalatinCoder** [LycheeOrg/Lychee#2469] [Bugs] [answered]
Hi Guys,
I'm wondering if it's possible to make a smart album ("starred" specifically) available for anonymous access? It seems to me that this was implemented in #293. However I'm not sure if this functionality got removed or if I misunderstood it.

cheers

**Comment by ildyria** [comment]
That should still be possible.

**Reply by ildyria** [reply]
But indeed it is broken. Lemme fix this.

**Reply by ildyria** [reply, marked as answer]
Fix is on its way. See #2470

**Reply by PalatinCoder** [reply]
Well that was quick 😉 I‘ll give it a try later today. Thanks!

**Reply by PalatinCoder** [reply]
Hi, I'm sorry it took me two weeks :/ I gave it a try today and it doesn't quite work as I expected. The option to make the starred album public is back again and working, as in "the album itself can be seen anonymously". However, I was under the impression that all starred pictures would then be visible too, which is not the case. The are only visible in the starred album if the "originating" album of the picture is public, too. That kind of defeats the purpose of selectively making certain pictures public. Is this the intended behavior and it's my use case that is wrong?

There is an alternative approach to my problem, which is to copy the pictures I want to be public to a second, public album. But starring them would be more comfortable ;-)

**Reply by ildyria** [reply]
> The are only visible in the starred album if the "originating" album of the picture is public, too.
> Is this the intended behavior?

Yes.

What you want is to be able to selectively make some pictures public. Unfortunately this functionality is gone (yeah sorry).
It was available in the past, but with the refactoring of the album rights to improve collaboration (e.g. you can now share an album with someone and give them upload rights). The complexity of making some pictures selectively public was increasing the complexity unnecessarily. In that regard the benefits of giving upload rights etc completely outweigh the benefits of making single pictures public and the awkward way of sharing them.

So yes, I guess the alternative approach to your problem will be the solution you are looking for.

**Reply by PalatinCoder** [reply]
Alright, as I said copying the pictures is still a solution and fits my use case. Thanks for clarifying!
